Apps/Contoso.UWP.Puppet/App.xaml.cs

```diff
@@ -37,6 +37,11 @@ public App()
             Suspending += OnSuspending;
             AppCenter.LogLevel = LogLevel.Verbose;
             AppCenter.SetLogUrl("https://in-integration.dev.avalanch.es");
+            object storageSize;
+            if (Windows.Storage.ApplicationData.Current.LocalSettings.Values.TryGetValue("StorageMaxSize", out storageSize))
```

Comment:
Should we use some prefix?

Reply:
It is not necessary in test applications.